ui/src/app/services/workspace-storage.service.ts

```diff
@@ -10,27 +12,32 @@ export interface WorkspaceData extends Workspace {
   accessLevel: WorkspaceAccessLevel;
 }
 
+const nextCurrentWorkspace = (workspace: WorkspaceData) => {
```

Comment:
nit: wrapping this in a helper method saves a few characters at callsites, but adds a layer of indirection for the reader. This is not a good tradeoff in my view - would prefer just inlining.

Reply:
I can just inline it, since it is relatively the same. (As compared to the ProfileStorageService which needs to pass some boilerplate)